src/api/config/routes.rb

```diff
@@ -287,6 +287,10 @@ def self.public_or_about_path?(request)
       post 'project/remove_person/:project' => :remove_person, constraints: cons
       post 'project/remove_group/:project' => :remove_group, constraints: cons
       get 'project/monitor/:project' => :monitor, constraints: cons, as: 'project_monitor'
+      # For backward compatibility
+      get 'project/monitor', to: redirect { |_path_parameters, request|
+        "/project/monitor/#{request.query_parameters[:project]}?#{request.query_parameters.except(:project).to_param}"
```

Comment:
This leaves an extra `?` when there are no query parameters beside `project`. So if you go to `project/monitor?project=home:Admin`, you are redirected to `/project/monitor/home:Admin?`

Reply:
Thanks. Fixed. Please have another look.